packages/platform-graphql/src/graphql-interceptor.service.ts

```diff
@@ -14,7 +14,8 @@
     return GqlExecutionContext.create(context);
   }
   getRequest(context: ExecutionContext): Request {
-    return this.getContext(context).getContext().req;
+    // For subscriptions, we need to get the connection from the context, not the req.
+    return this.getContext(context).getContext().req || this.getContext(context).getContext().connection;
```

Comment:
Codacy has a fix for the issue: [Replace `this.getContext(context).getContext().req·||·this.getContext(context).getContext().connection` with `(⏎······this.getContext(context).getContext().req·||·this.getContext(context).getContext().connection⏎····)`](https://app.codacy.com/gh/jmcdo29/ogma/pullRequest?prid=8963871)
```suggestion
    return (
```